docs/Learning-Environment-Create-New.md

```diff
@@ -472,3 +472,18 @@ RollerBall environment:
    your Assets browser, turning it into a prefab.
 1. You can now instantiate copies of the TrainingArea prefab. Drag them into
    your scene, positioning them so that they do not overlap.
+
+## Optional: Training Using Concurrent Unity Instances
+Another level of parallelization comes by training using
+[concurrent Unity instances](https://github.com/Unity-Technologies/ml-agents/blob/main/docs/ML-Agents-Overview.md#additional-features).
```

Comment:
can you make this a relative path instead of an absolute? so as we create different releases, this path points to the appropriate (relative) file.

Reply:
Done.